src/Digdir.Domain.Dialogporten.Application/Common/Numbers/EndUserIdentifier.cs

```diff
@@ -0,0 +1,52 @@
+using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace Digdir.Domain.Dialogporten.Application.Common.Numbers;
+
+internal static partial class EndUserIdentifier
+{
+    private static readonly int[] NorwegianIdentifierNumberWeights1 = [3, 7, 6, 1, 8, 9, 4, 5, 2, 1];
+    private static readonly int[] NorwegianIdentifierNumberWeights2 = [5, 4, 3, 2, 7, 6, 5, 4, 3, 2, 1];
+
+    [GeneratedRegex(@"urn:altinn:([\w-]{5,20}):([\w-]{4,20})::([\w-]{11,36})", RegexOptions.None, matchTimeoutMilliseconds: 100)]
```

Comment:
```suggestion
    [GeneratedRegex(@"urn:altinn:([\w-]{5,20}):([\w-]{4,20})::([\w-]{5,36})", RegexOptions.None, matchTimeoutMilliseconds: 100)]
```

Vi vet lite om hvordan en enduser-id vil se ut på dette stadiet, men det kan nok fort tenkes at det er mindre enn 11 tegn i noen sammenhenger, så 5-36 alphanum pluss bindestrek tror jeg er et passe greit utgangspunkt.